client/package.json

```diff
@@ -5,7 +5,7 @@
   "version": "0.0.0",
   "type": "module",
   "scripts": {
-    "dev": "vite",
+    "dev": "vite --host",
```

Comment:
this is very nice, makes all the other api calls cleaner.

Reply:
It's actually the changes to `vite.config.ts` that let us remove that old env variable we had before. This change lets you run the app on another device (e.g. your phone) by navigating to the IP address that Vite spites out when you spin up the dev server. I will have to probably update the README with instructions for this, since it's useful for developing for mobile.